vendor/engines/secure_rooms/app/views/secure_rooms/indala_numbers/edit.html.haml

```diff
@@ -0,0 +1,17 @@
+= content_for :head_content do
+  = render "users/form_head"
```

Comment:
`users/form_head` is just setting up some JS for keeping username and email address synced up as you type. Is it necessary here?